devlib/target.py

```diff
@@ -1041,6 +1041,12 @@ def adb_reboot_bootloader(self):
     def adb_root(self):
         adb_command(self.adb_name, 'root', timeout=30)
 
+    def usb_charge(self, enable=True, sysfs_path=None):
```

Comment:
Why is the path being passed as argument? Given that it is in sysfs, I would expect it to be standardized.

Even if it can vary, it is not something I would expect the calling code to be aware of. If there are several common possibilities (e.g. varying between kernel versions etc), they should all be tested in turn, starting with the most likely. If the path can vary widely between different devices, it should be added as a configurable option in the `Platform` (which encapsulates device-specific knowledge).